coverage/client.test.ts

```diff
@@ -13,15 +13,15 @@ describe('Client', () => {
     Client.get('cards', params)
       .then(response => {
         expect(response).to.be.a('array');
-        expect(response[0].name).to.equal('Gardevoir');
+        expect(response[0].name).to.equal('Ampharos');
```

Comment:
This seems like a bad test. Perhaps it could check to see if it got a _nonzero_ amount of cards back?

Reply:
Yeah, something like this?

```typescript
expect(response.length).to.have.lengthOf.above(0);
```